src/Compilers/CSharp/Portable/Syntax/CSharpSyntaxWalker.cs

```diff
@@ -36,10 +36,11 @@ public override void DefaultVisit(SyntaxNode node)
         {
             var childCnt = node.ChildNodesAndTokens().Count;
             int i = 0;
+            var slotData = new ChildSyntaxList.SlotData(slotIndex: 0, precedingOccupantSlotCount: 0, node.Position);
```

Comment:
Nit: consider having a special constructor (or factory method) for this that takes just `node`.

Reply:
I like it, done!